src/schematools/contrib/django/factories.py

```diff
@@ -167,20 +170,13 @@ def _get_related_name(self):
             # as Django still uses the backwards relation internally.
             return "+"
 
-    @property
-    def field_kwargs(self):
-        # In schema foreign keys should be specified without _id,
-        # but the db_column should be with _id
-        kwargs = {
-            **super().field_kwargs,
-            "db_column": f"{to_snake_case(self.field.name)}_id",
-            "db_constraint": False,
-            "related_name": self._get_related_name(),
-        }
+    def _get_to_field(self) -> Optional[str]:
+        """Tell whether the foreign key doesn't point to the targets's primary key.
```

Comment:
This suggests that this a predicate, but the return value is a string.

Reply:
ok, updated!